tailwind.config.js

```diff
@@ -4,7 +4,34 @@ module.exports = {
     "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
```

Comment:
We don't want to use tailwind colors. Will this override them or append to them?

Reply:
It only append our colors, but if we're not going to use the default colors, I can disable them